Dockerfile

```
@@ -0,0 +1,14 @@
+ARG GOVERSION=latest
+FROM golang:$GOVERSION AS builder
+
+WORKDIR /src
+COPY . .
+
+RUN GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o . -ldflags="-s -w -X main.release=$(git rev-parse HEAD)" .
+
+FROM alpine
```

```suggestion
FROM alpine:3
```
...to avoid at least some surprises when they bump the major version.